packages/plugin-kit/src/hooks/slice-update.ts

```diff
@@ -12,12 +14,16 @@ import type {
 export type SliceUpdateHookData = {
 	libraryID: string;
 	model: SharedSliceModel;
+	mocks?: SharedSliceContent[];
```

Comment:
Why do mocks have to be passed to `slice:update` hook? This sounds unneeded/introducing a side effect/business logic to the hook